README.md

```diff
@@ -135,6 +135,49 @@ AnyCable, k6, WebSockets, and Yabeda](https://evilmartians.com/chronicles/real-t
+- [xk6-output-kafka](https://github.com/grafana/xk6-output-kafka) - Export k6 results in real-time to Kafka.
+- [xk6-output-timescaledb](https://github.com/grafana/xk6-output-timescaledb) - Export k6 results to TimescaleDB.
+- [xk6-sql](https://github.com/grafana/xk6-sql) - Load-test SQL Servers (PostgreSQL, MySQL and SQLite3 for now).
+- [xk6-ssh](https://github.com/grafana/xk6-ssh) - SSH.
+
+### Community
+
+- [xk6-cable](https://github.com/anycable/xk6-cable) - Test Action Cable and AnyCable functionality.
+- [xk6-client-prometheus-remote](https://github.com/grafana/xk6-client-prometheus-remote) - Test Prometheus Remote Write performance.
+- [xk6-coap](https://github.com/golioth/xk6-coap) - Interact with Constrained Application Protocol endpoints.
+- [xk6-dotenv](https://github.com/szkiba/xk6-dotenv) - Load env vars from a .env file.
+- [xk6-ethereum](https://github.com/distribworks/xk6-ethereum) - K6 extension for ethereum protocols.
+- [xk6-faker](https://github.com/szkiba/xk6-faker) - Generate random fake data.
+- [xk6-file](https://github.com/avitalique/xk6-file) - Write files.
+- [xk6-g0](https://github.com/szkiba/xk6-g0) - Write k6 tests in golang.
+- [xk6-kafka](https://github.com/mostafa/xk6-kafka) - Load-test Apache Kafka. Includes support for Avro messages.
+- [xk6-kv](https://github.com/oleiade/xk6-kv) - Share key-value data between VUs.
+- [xk6-mock](https://github.com/szkiba/xk6-mock) - Mock HTTP(S) servers.
+- [xk6-mqtt](https://github.com/pmalhaire/xk6-mqtt) - MQTT extension.
+- [xk6-nats](https://github.com/ydarias/xk6-nats) - Provides NATS support for k6 tests.
+- [xk6-opentelemetry](https://github.com/thmshmm/xk6-opentelemetry) - Generate OpenTelemetry signals from within your test scripts.
+- [xk6-output-elasticsearch](https://github.com/elastic/xk6-output-elasticsearch) - Export results to Elasticsearch 8.x.
+- [xk6-output-prometheus-pushgateway](https://github.com/martymarron/xk6-output-prometheus-pushgateway) - Export results to Prometheus pushgateway.
+- [xk6-output-statsd](https://github.com/LeonAdato/xk6-output-statsd) - Enables real-time output of test metrics to a StatsD service.
+- [xk6-output-timestream](https://github.com/leonyork/xk6-output-timestream) - Export results to AWS Timestream.
+- [xk6-playwright](https://github.com/nicholasvuono/xk6-playwright) - Browser automation and end-to-end web testing using Playwright.
+- [xk6-prometheus](https://github.com/szkiba/xk6-prometheus) - Prometheus HTTP exporter for k6.
+- [xk6-prompt](https://github.com/Juandavi1/xk6-prompt) - Support for input arguments via UI.
+- [xk6-sse](https://github.com/phymbert/xk6-sse) - A k6 extension for Server-Sent Events (SSE).
+- [xk6-tcp](https://github.com/NAlexandrov/xk6-tcp) - Send data to TCP port.
+- [xk6-top](https://github.com/szkiba/xk6-top) - Updating the current k6 metrics summaries on the terminal during the test run.
+- [xk6-ts](https://github.com/grafana/xk6-ts) - Add TypeScript support for k6.
```

Comment:
Is this extension still needed after k6 [got TS support in v0.52.0](https://github.com/grafana/k6/pull/3738)?

Reply:
You are right, I will deprecate the extension and delete it from here.
Its functionality is not quite the same, but I don't think it's worth maintaining.